src/infrastructure/api/index.ts

```diff
@@ -0,0 +1,47 @@
+import express from 'express';
+
+const app = express();
+
+const router = express.Router();
+
+const { API_VERSION } = process.env;
+
+// Declare middlewares
+/**
+ * bodyParser, error handling, logger, etc..
+ * http://expressjs.com/en/starter/basic-routing.html
+ * http://expressjs.com/en/guide/using-middleware.html
+ */
+
+/**
+ * app level middlewares
+ * app.use('/path', (req, res, next) => {
+ * chain middlewares
+ * next()
+ * })
+ */
+app.use((req, __, next) => {
```

Comment:
Something we can also discuss about how we want to document our open api specs so that APIG can use it to be configured but another rationale (in addition to the proxy integration currently used in CVS) was that the code should be self documenting and was thinking that we could leverage the following tool to generate our specs (from the code).
https://github.com/Surnet/swagger-jsdoc
https://github.com/scottie1984/swagger-ui-express

Reply:
Set up swagger-jsdoc to create swagger.yaml.

Reply:
This was causing issues, so it's been put on hold for now so that VOTT can continue.